src/d3a/d3a_core/util.py

```diff
@@ -402,3 +402,13 @@ def convert_datetime_to_str_keys(indict, outdict):
                 convert_datetime_to_str_keys(indict[key], outdict[key])
 
     return outdict
+
+
+def get_number_of_unmatched_loads(indict):
```

Comment:
This function is too specific to be placed in a generic module like utils. Can it not be located in the unmatched loads modules ?

Reply:
Yes, you are right. will move it there.